public/app/features/alerting/unified/components/rules/state-history/useRuleHistoryRecords.tsx

```diff
@@ -153,3 +195,132 @@ export function logRecordsToDataFrame(
 
   return frame;
 }
+
+type Data = {
```

Comment:
This type is a bit opaque. Could we improve the naming there? We could also change it from `type` to `interface` for better TS performance and consistency with the codebase